app/helpers/fuzzy_time.rb

```diff
@@ -24,7 +24,7 @@ def ago(timestamp)
                      end
           "<span data-toggle='tooltip' title='#{timestamp.strftime('%e %B %Y at %H:%M %Z')}'>#{response}</span>"
```

Comment:
It _does_ make sense to localize "everything", while we're localizing stuff... I can do it if you'd like it @kytrinyx!